functions/vcs.zsh

```diff
@@ -110,7 +110,18 @@ function +vi-hg-bookmarks() {
 
 function +vi-vcs-detect-changes() {
   if [[ "${hook_com[vcs]}" == "git" ]]; then
-    vcs_visual_identifier='VCS_GIT_ICON'
+
+    local remote=`git ls-remote --get-url`
+    if [[ "$remote" =~ "github" ]] then
```

Comment:
@dritter thanks for the good explanation. I understand better now. I think is fine to leave it as it is then, in any case a refactoring would improve the cyclomatic complexity only which is not so much actually.